### PR TITLE
[2201.8.x] Update bal deprecate command help

### PR DESCRIPTION
**Pull request by Dilhasha**
Fixes part of https://github.com/ballerina-platform/ballerina-dev-website/issues/8601

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [x] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by ShammiL — APPROVED** [review]

**Review by gayaldassanayake — APPROVED** [review]

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help [hunk 1]
```diff
@@ -2,28 +2,40 @@ NAME
...
-       bal deprecate [OPTIONS] <package>
+       bal deprecate [OPTIONS] <org-name>/<package-name>[:<version>]
 
 
 DESCRIPTION
-       The deprecate command marks the specified package as deprecated in Ballerina central.
+       Deprecate a package published in Ballerina central.
+
+       If a specific version of a package is provided, that version will be deprecated.
+       Alternatively, if no version is specified, all versions of the package will be deprecated.
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help:12]:
```suggestion
       If no version is specified, all versions of the package will be deprecated.
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help [hunk 2]
```diff
@@ -2,28 +2,40 @@ NAME
...
-       This command does not delete the package from Ballerina central, and the package can be undeprecated using the `--undo` option.
+       This command does not delete the package from Ballerina central, and the package deprecation can be undone using the `--undo` option.
 
 
 OPTIONS
        --message=<msg>
            Use the given <msg> as the deprecation message
        --undo
-           Undeprecate a deprecated package
+           Undo deprecation of package
```
Comment (azinneera) [inline cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help:27]:
```suggestion
           Undo deprecation of a package
```

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help (hunk 1 above)
Comment (Dilhasha) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help]: Fixed with 607da826707c9eddb8e331e0f2c147bb60b862f5

File: cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help (hunk 2 above)
Comment (Dilhasha) [reply cli/ballerina-cli/src/main/resources/cli-help/ballerina-deprecate.help]: Fixed with 607da826707c9eddb8e331e0f2c147bb60b862f5

**Review by azinneera — APPROVED** [review]

**Pull request merged**